shared/js/background/chrome-events.es6.js

```diff
@@ -56,10 +64,30 @@ chrome.webRequest.onHeadersReceived.addListener(
             // returns a promise
             return ATB.updateSetAtb(request)
         }
+
+        let responseHeaders = request.responseHeaders
+        const activeExperiment = settings.getSetting('activeExperiment')
+        if (activeExperiment) {
+            const experiment = settings.getSetting('experimentData')
+
+            if (experiment && experiment.blockingActivated) {
```

Comment:
As this block comes up 3 times in this file, could we extract this into a function that just returns if we're in the cookie blocking experiment?